src/extensionVariables.ts

```diff
@@ -17,6 +18,7 @@ export namespace ext {
     export let context: ExtensionContext;
     export let outputChannel: OutputChannel;
     export let reporter: ITelemetryReporter;
+    export let resourcesPath = path.join(__dirname, 'resources');
```

Comment:
You have `resourcesPath` in both constants and extensionVariables. Do we need both? Seems kinda confusing.

Reply:
Removed from extvars